reference/commands/graph/info.rst

```diff
@@ -356,9 +350,340 @@ For example, to get the options of zlib, the following command could be run:
+                "test": false
+            },
+            "2": {
+                "ref": "zlib/1.2.13#13c96f538b52e1600c40b88994de240f",
+                "id": "2",
+                "recipe": "Cache",
+                "package_id": "d0599452a426a161e02a297c6e0c5070f99b4909",
+                "prev": "69b9ece1cce8bc302b69159b4d437acd",
+                "build_id": null,
+                "binary": "Skip",
+                "invalid_build": false,
+                "info_invalid": null,
+                "name": "zlib",
+                "user": null,
+                "channel": null,
+                "url": "https://github.com/conan-io/conan-center-index",
+                "license": "Zlib",
+                "author": null,
+                "description": "A Massively Spiffy Yet Delicately Unobtrusive Compression Library (Also Free, Not to Mention Unencumbered by Patents)",
+                "homepage": "https://zlib.net",
+                "build_policy": null,
+                "upload_policy": null,
+                "revision_mode": "hash",
+                "provides": null,
+                "deprecated": null,
+                "win_bash": null,
+                "win_bash_run": null,
+                "default_options": {
+                    "shared": false,
+                    "fPIC": true
+                },
+                "options_description": null,
+                "version": "1.2.13",
+                "topics": [
+                    "zlib",
+                    "compression"
+                ],
+                "package_type": "static-library",
+                "settings": {
+                    "os": "Macos",
+                    "arch": "x86_64",
+                    "compiler": "apple-clang",
+                    "compiler.version": "12.0",
+                    "build_type": "Release"
+                },
+                "options": {
+                    "fPIC": "True",
+                    "shared": "False"
+                },
+                "options_definitions": {
+                    "shared": [
+                        "True",
+                        "False"
+                    ],
+                    "fPIC": [
+                        "True",
+                        "False"
+                    ]
+                },
+                "generators": [],
+                "system_requires": {},
+                "recipe_folder": "/Users/franchuti/.conan2/p/zlib9f370ca971ddf/e",
+                "source_folder": null,
+                "build_folder": null,
+                "generators_folder": null,
+                "package_folder": null,
+                "cpp_info": {
+                    "root": {...}
+                },
+                "label": "zlib/1.2.13",
+                "dependencies": {},
+                "context": "host",
+                "test": false
+            }
+        },
+        "root": {
+            "0": "None"
+        },
+        "overrides": {},
+        "resolved_ranges": {}
+    }
```

Comment:
Full output or should I collapse more dicts?

Reply:
As the format is the same for several commands, like create, install, graph... I think we should make a separate section to document the format and link it in each of the commands?

Reply:
I agree, there should be a section with this json file format, that is linked by all the different commands.